stable/prometheus/templates/server-deployment.yaml

```diff
@@ -87,7 +87,7 @@ spec:
           {{- end }}
           args:
           {{- if .Values.server.retention }}
-            - --storage.tsdb.retention={{ .Values.server.retention }}
+            - --storage.tsdb.retention.time={{ .Values.server.retention }}
```

Comment:
is there any need to update the readme: https://github.com/helm/charts/blob/master/stable/prometheus/README.md (Search for storage.local.retention)

Reply:
I can't evaluate this because i don't know if this property is relevant for the migration.
If it is false, then it is false for a longer time, because in the README is a completly different name (storage.**local**.retention).